devimint/Cargo.toml

```diff
@@ -17,6 +17,10 @@ axum = { version = "0.7.5", features = ["tracing"] }
 bitcoincore-rpc = { workspace = true }
 clap = { version = "4.5.7", features = ["derive", "env", "std", "help", "usage", "error-context", "suggestions"], default-features = false }
 cln-rpc = { workspace = true }
+esplora-client = { version = "0.6.0", default-features = false, features = [
```

Comment:
If we are using it elsewhere already, can we make it a workspace dep?

Reply:
Done.